media-source/mp4/test-v-no-keyframe-segment-manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+    "url": "mp4/mp4-main-ogop-h264ogop_mid1.mp4",
```

Comment:
Why not just have a single mp4 for the mp4 variant of this new test, that is a concatenation of the init plus mid1? That's how it's used in the test.